ccdproc/combiner.py

```diff
@@ -177,6 +177,58 @@ def scaling(self, value):
             # reshape so that broadcasting occurs properly
             self._scaling = self.scaling[:, np.newaxis, np.newaxis]
 
+    # set up IRAF-like minmax clipping
+    def iraf_minmax_clipping(self, nlow=None, nhigh=None):
```

Comment:
I debated this.  My feeling is that I'd rather default to 0 or None (no rejection) because (thinking as a user) if I call this and only specify nhigh=1 with no call to nlow, I would expect one value to be rejected, not two.

I prefer to leave the None check in because I could see a user specifying that to try to turn off low or high rejection.  I think it makes more sense for the defaults should be 0, not None though, so I'll change that.